update.sh

```diff
@@ -0,0 +1,4 @@
+cd "/c/Users/emelwa/Desktop/Code/Projects/Puzzle-Game/"
+git pull git@github.com:mellepella/Puzzle-Game.git
+echo Updated. Press enter to exit...
+read
```

Comment:
Haha nice script xD

Reply:
Yep, I had some people not familiar with git trying out the game😎 (and they did not want to download it everytime a new level comes)

Reply:
Realized it does not work on any computer except mine... Oops...